### ISSUE TITLE
Flake: hack/build-images.sh fails with "stat examples/gitserver/bin/gitserver: no such file or directory"

### ISSUE THREAD

**Issue by tnozicka** [openshift/origin#17684]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/17620/test_pull_request_origin_extended_conformance_install/3776/

```
[WARNING] Retrying image build for openshift/origin-gitserver:latest, attempt 1...
[openshift/origin-gitserver] --> FROM openshift/origin
[openshift/origin-gitserver] --> COPY bin/gitserver /usr/bin/gitserver
[openshift/origin-gitserver] stat examples/gitserver/bin/gitserver: no such file or directory
```

**Comment by php-coder** [comment]
Hm.. it should be fixed already (https://github.com/openshift/origin/issues/17345).

CC @jupierce

**Comment by tnozicka** [comment]
this PR was for release-3.7 branch and it doesn't have https://github.com/openshift/origin/pull/17373

**Comment by jupierce** [comment]
@tnozicka The release-3.7 branch is no longer pulled into OCP 3.7 and should not be targeted by PRs. You will need to PR against ose/enterprise-3.7 .

**Comment by tnozicka** [comment]
@jupierce what if I want to fix origin as well, why shouldn't I target release-3.7? 3.7 is our latest stable release in upstream.

I am not sure why we don't auto pull stuff from origin release-3.7 anymore but I'll send it manually there too.

**Comment by jupierce** [comment]
@tnozicka We just don't offer long term support in these branches.

**Comment by tnozicka** [comment]
If previous releases in origin are of any guidance we used to release x.x.1 at least. I went though the history and we've released (1.1.6, 1.2.2, 1.3.3, 1.4.1, 1.5.1, 3.6.1). 

@smarterclayton is 3.7 in origin really dead? Are we sure there won't be 3.7.1? For one I know that 3.7.0 `oc cluster up` is completely broken on some platforms (https://github.com/openshift/origin/issues/17638)

How can we mark 3.7 as not supported without having a newer release?

**Comment by smarterclayton** [comment]
The branch should be fixed, but gitserver should be removed.

**Comment by smarterclayton** [comment]
We ensure oc cluster up works on origin release branches, and occasionally
do security fixes.  So we should fix things broken on the branch that would
make cutting a release for security harder.

On Dec 21, 2017, at 12:30 PM, Tomáš Nožička <notifications@github.com>
wrote:

If previous releases in origin are of any guidance we used to release x.x.1
at least. I went though the history and we've released (1.1.6, 1.2.2,
1.3.3, 1.4.1, 1.5.1, 3.6.1).

@smarterclayton <https://github.com/smarterclayton> is 3.7 in origin really
dead? Are we sure there won't be 3.7.1? For one I know that 3.7.0 oc
cluster up is completely broken on some platforms (#17638
<https://github.com/openshift/origin/issues/17638>)

How can we mark 3.7 as not supported without having a newer release?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/issues/17684#issuecomment-353409920>,
or mute the thread
